src/palette/view/widgets/timesignaturepropertiesdialog.cpp

```diff
@@ -211,11 +211,17 @@ void TimeSignaturePropertiesDialog::accept()
 
     notation->undoStack()->prepareChanges();
 
-    m_originTimeSig->undoChangeProperty(Pid::TIMESIG_TYPE, int(m_editedTimeSig->timeSigType()));
-    m_originTimeSig->undoChangeProperty(Pid::SHOW_COURTESY, m_editedTimeSig->showCourtesySig());
-    m_originTimeSig->undoChangeProperty(Pid::NUMERATOR_STRING, m_editedTimeSig->numeratorString());
-    m_originTimeSig->undoChangeProperty(Pid::DENOMINATOR_STRING, m_editedTimeSig->denominatorString());
-    m_originTimeSig->undoChangeProperty(Pid::GROUP_NODES, g.nodes());
+    // Change linked mmr timesigs too
+    for (EngravingObject* obj : m_originTimeSig->linkList()) {
+        TimeSig* timeSig = toTimeSig(obj);
+        if (timeSig == m_originTimeSig || (timeSig->track() == m_originTimeSig->track() && timeSig->score() == m_originTimeSig->score())) {
+            timeSig->undoChangeProperty(Pid::TIMESIG_TYPE, int(m_editedTimeSig->timeSigType()));
```

Comment:
As far as I know, undoChangeProperty changes properties for linked objects automatically. So I wonder why it doesn't work in this case...